packages/cdc-react/package.json

```diff
@@ -53,6 +53,7 @@
     "vitest": "^0.33.0"
   },
   "dependencies": {
-    "@uswds/uswds": "^3.5.0"
+    "@uswds/uswds": "^3.5.0",
+    "react-router-dom": "^6.15.0"
```

Comment:
Please remove this dependency and any imports from it.

Reply:
Good call out! Thanks!